src/_includes/layouts/base.html

```diff
@@ -65,12 +65,13 @@
     <link rel="preconnect" href="https://www.googletagmanager.com/">
 
     <script>
-        (function() {
-            var theme = window.localStorage.getItem("theme");
-            if (!theme && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-                document.documentElement.setAttribute('data-theme', 'dark');
-            } else if (theme) document.documentElement.setAttribute('data-theme', theme);
-        })();
+            (function () {
+                var theme = window.localStorage.getItem("theme");
+                if (theme) document.documentElement.setAttribute('data-theme', theme)
+                else if (!theme && window.matchMedia('(prefers-color-scheme: dark)').matches)
```

Comment:
We have already filtered if theme is true in the first condition and we can expect it to be false in the next all conditions ?
```suggestion
                else if (window.matchMedia('(prefers-color-scheme: dark)').matches)
```

Reply:
yes we can remove the theme check.